### DISCUSSION TITLE
Enabling NVIDIA GPU in NC AIO with Windows Docker Desktop (WDD) Problems

### DISCUSSION THREAD

**Discussion by FlattusBlastus** [nextcloud/all-in-one#2267] [Questions] [answered]
Adding this to my docker compose file caused WDD to FREAK OUT:
    deploy:
        resources:
            reservations:
                devices:
                    - driver: nvidia
                      count: 1
                      capabilities: [gpu]
I had to do all kinds of things to get WDD to behave again. Weirdly enough, I was able to make it work on the linuxserver.io/plex image. I think it has something to do with Alpine maybe?
set DOCKER_BUILDKIT=  to one or zero then rebooting didnt seem to make any difference.

**Comment by FlattusBlastus** [comment]
Most of my other containers are now corrupt. Leave DOCKER_BUILDKIT= alone!

**Comment by szaimen** [comment, marked as answer]
This is unrelated to AIO.
